static/scss/partials/main.scss

```diff
@@ -1013,6 +1013,12 @@ firefox-private-relay-addon[data-addon-installed="true"] + firefox-private-relay
     display: none;
 }
 
+
+/* Show the opt in data notification banner if add on is downloaded */
+firefox-private-relay-addon[data-addon-installed="false"] + firefox-private-relay-addon-data + main .c-data-notification-banner {
```

Comment:
As I understand it, this hides the banner if the add-on is not installed? Maybe we could use a different class (e.g. `.c-notification-banner`) for the styling, and this line to only hide `.c-data-notification-banner` specifically, so that it can be re-used in other settings (thinking of the settings screen in particular, of course).

Reply:
@Vinnl May need your help adding logic for that part, since I'm not super sure how it's synced up to the settings logic you built